### PR TITLE
[bitnami/vault] Enabling Vault run in one single Namepace and with that, add more tha…

### PR DESCRIPTION
**Pull request by matheusmgon**
Small change to able run multiples Bitnami Vault helm chart in one single Cluster.

### CODE REVIEW

File: bitnami/vault/templates/injector/mutating-webhook.yaml [hunk 1]
```diff
@@ -40,4 +40,6 @@ webhooks:
           operator: NotIn
           values:
             - injector
+    namespaceSelector:
+{{ tpl (toYaml (((.Values.injector.webhook)).namespaceSelector | default .Values.injector.namespaceSelector)) . | indent 6}}
```
Comment (migruiz4) [inline bitnami/vault/templates/injector/mutating-webhook.yaml:44]: I wouls like to suggest using the helper `common.tplvalues.render` in combination with 
```suggestion
    namespaceSelector: {{- include "common.tplvalues.render" (dict "value" (default .Values.injector.namespaceSelector ((.Values.injector.webhook)).namespaceSelector) "context" $) | nindent 6 }}

```

Comment (migruiz4) [reply bitnami/vault/templates/injector/mutating-webhook.yaml:44]: Additionally, the value `injector.namespaceSelector` does not exist in the values.yaml.

Is there a reason to have both `injector.webhook.namespaceSelector` and `injector.namespaceSelector`?

**Review by migruiz4 — CHANGES_REQUESTED** [review]
Hi @matheusmgon,

Thank you very much for your contribution! Could you please take a look at my comments?

File: bitnami/vault/templates/injector/mutating-webhook.yaml (hunk 1 above)
Comment (matheusmgon) [reply bitnami/vault/templates/injector/mutating-webhook.yaml]: There are no reason to add `injector.namespaceSelector`, it was a mistake.
Now I add the way like you sent and plus I add a condition to check if this info was sent.
Just to be sure, nothing else.

**Comment by matheusmgon** [comment]
I really don't know why license file is failing 🤔

File: bitnami/vault/values.yaml [hunk 2]
```diff
@@ -1387,6 +1387,18 @@ injector:
...
+  ## Example:
+  ## webhook:
+  ##   namespaceSelector:
+  ##     matchExpressions:
+  ##     - key: kubernetes.io/metadata.name
+  ##       operator: In
+  ##       values:
+  ##         - "{{ .Release.Namespace }}"
+  webhook: {}
+
```
Comment (migruiz4) [inline bitnami/vault/values.yaml:1401]: Please use the following structure:

```suggestion
  webhook: 
    ## @param injector.webhook Enabling specify which namespace you want to work with webhook
    ## Example:
    ## webhook:
    ##   namespaceSelector:
    ##     matchExpressions:
    ##     - key: kubernetes.io/metadata.name
    ##       operator: In
    ##       values:
    ##         - "{{ .Release.Namespace }}"
    namespaceSelector: {}
```

This way users should not be confused about what `webhook.*` values can be configured or not, and leaves room for new values to be added in the future.

**Review by migruiz4 — COMMENTED** [review]
Please take a look at the new comments

File: bitnami/vault/values.yaml (hunk 2 above)
Comment (matheusmgon) [reply bitnami/vault/values.yaml]: Someting failed during the creation of the Readme.md
Do you know why @migruiz4 ?

**Review by migruiz4 — COMMENTED** [review]
Looks like the branch is all messed up, could you please fix it?

I would recommend recreating it or creating a new PR if necessary

File: bitnami/vault/values.yaml (hunk 2 above)
Comment (migruiz4) [reply bitnami/vault/values.yaml]: My bad, I fixed it in the suggestion above.

The README is autogenerated based on the `@param` tag, which was missing `.namespaceSelector`.
It should be `@param injector.webhook.namespaceSelector`

**Comment by migruiz4** [comment]
Thank you for creating #25629, we will continue there

**Pull request closed without merging**